packages/mobile/src/identity/verification.ts

```diff
@@ -211,13 +222,23 @@ export function* doVerificationFlow() {
     return true
   } catch (error) {
     Logger.error(TAG, 'Error occured during verification flow', error)
+    error.message === ErrorMessages.INSUFFICIENT_BALANCE
```

Comment:
not to be too pedantic, but it feels a little weird to have this logic in the error handling - it's not really an "error" in the sense that we know exactly why the user is unable to complete verification. i wonder if just moving the put to your `if (!balanceIsSufficient)` block and returning there would be sufficient instead of throwing